collectors/ebpf.plugin/ebpf_dcstat.c

```diff
@@ -159,9 +159,9 @@ void ebpf_dcstat_create_apps_charts(struct ebpf_module *em, void *ptr)
 
     ebpf_create_charts_on_apps(NETDATA_DC_REFERENCE_CHART,
                                "Count file access.",
-                               EBPF_CACHESTAT_DIMENSION_HITS,
+                               EBPF_COMMON_DIMENSION_FILES,
                                NETDATA_APPS_DCSTAT_GROUP,
-                               NETDATA_EBPF_CHART_TYPE_STACKED,
+                               NETDATA_EBPF_CHART_TYPE_LINE,
```

Comment:
Why did you changed that?

Reply:
`STACKED` was correct.

Reply:
Sorry my fault, I changed in the wrong branch a test I was doing. :/